examples/jsm/webxr/XRHandModelFactory.d.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	Group,
+	Object3D,
+} from '../../../src/Three';
+
+import { GLTFLoader } from '../loaders/GLTFLoader';
+
+export class XRHandModel extends Object3D {
+
+	constructor();
+
+	motionController: any;
+
+}
+
+export class XRHandModelFactory {
+
+	constructor( gltfLoader?: GLTFLoader );
+	gltfLoader: GLTFLoader | null;
+	path: string;
+
+	createHandModel(
+		controller: Group,
+		profile?: 'spheres' | 'boxes' | 'oculus',
+		options?: { primitive: 'sphere' | 'box' }
```

Comment:
The `model` property is missing here which is part of the `options` of `XRHandOculusMeshModel`.

Reply:
Thanks. I added the `model` property to include `lowpoly`.